cloudbuild.yaml

```diff
@@ -4,17 +4,11 @@ steps:
                 - 'build'
                 - '--tag=%REPO%/mkdebootstrap:%VERSION%'
                 - 'mkdebootstrap/'
-        - name: gcr.io/cloud-builders/docker
-          args:
-                - 'build'
-                - '--tag=docker-mkdebootstrap'
-                - 'docker-mkdebootstrap'
-        # This is an intermediate step in order to run docker in privileged mode.
-        # It just pulls the mkdebootstrap image built just above and runs
-        # debootstrap on it.
-        - name: docker-mkdebootstrap
+        - name: %REPO%/mkdebootstrap:%VERSION%
           args:
-                - '%REPO%'
+                - '-d/workspace'
```

Comment:
adding a space adds a space to the actual path it passes to argo which causes it to fail